azure-gaming-pr/reference-architectures/unreal-pixel-streaming-deploying.md

```diff
@@ -52,22 +52,22 @@ Depending on your **Region** needs, choose a **Region** that is the closest to t
+13. In the Networking page on the left, click the [Add inbound port rule](/azure/virtual-machines/windows/nsg-quickstart-portal#create-an-inbound-security-rule) button to open up any additional ports such as 8888 (PixelStreamingPort between UE app + SS), 9999 (if using MM), 443 (if needing Https) and any STUN/TURN server ports being used (e.g., 19302). STUN/TURN servers not _required_ for most users, though some mobile or restricted networks might require these—learn more [here](#turn-and-stun-servers). As always, please follow your security requirements to ensure desired network safety.
 
 [![Open VM Ports](media/pixel-streaming/pixel-streaming-add-ports.png)](media/pixel-streaming/pixel-streaming-add-ports.png)
 
 14. Now you are ready to Remote Desktop into the VM in Azure. In the left **Overview** tab in the Azure Portal click the **Connect** button and choose **RDP** , and then click the **Download RDP File** button to save the connection file to your local computer. Follow the quick-start documentation [here](/azure/virtual-machines/windows/quick-create-portal#connect-to-virtual-machine) for more on connecting to the VM via RDP, and remember to use the same login/password set when creating the VM. **Note:** When opening the RDP file, choose **More choices** and **Use a different account** to type in the login and password (in case login defaults to your local account). Ensure the name / password previously chosen are correctly entered to login to the VM.
 
 [![RDP Login](media/pixel-streaming/pixel-streaming-rdp-login.png)](media/pixel-streaming/pixel-streaming-rdp-login.png)
 
-15. Once connected to the VM, accept the Windows Privacy options and copy (right click -&gt; copy) and paste (right click -&gt; paste) the exported Pixel Streaming package ([Unreal docs](https://docs.unrealengine.com/SharingAndReleasing/PixelStreaming/PixelStreamingIntro/index.html)) to the Windows desktop of the VM (i.e., WindowsNoEditor folder contents). **Note:** To speed up the copy it&#39;s recommended to zip up the files before copying over.
+15. Once connected to the VM, accept the Windows Privacy options and copy (right click -&gt; copy) and paste (right click -&gt; paste) the exported Pixel Streaming package ([Unreal docs](https://docs.unrealengine.com/SharingAndReleasing/PixelStreaming/PixelStreamingIntro/index.html)) to the Windows desktop of the VM (i.e., WindowsNoEditor folder contents for UE4 apps, or Windows folder for UE5). **Note:** To speed up the copy it&#39;s recommended to zip up the files before copying over.
```

Comment:
I know your change is just to update some details in the instructions.

Do you mean that developers should follow the instructions on this page to create the package?: [Unreal docs](https://docs.unrealengine.com/SharingAndReleasing/PixelStreaming/PixelStreamingIntro/index.html)

Reply:
Yes, I put this pre-requisite up at the top as well to make it more clear.